cst.yml

```
@@ -5,7 +5,7 @@ metadataTest:
       value: "/home/helm/.helm"
   labels:
     - key: io.jenkins-infra.tools
-      value: "helm,kubectl,helmfile,sops,aws-cli,aws-iam-authenticator,yamllint,updatecli,jenkins-agent,doctl,jq,yq,ruby"
+      value: "helm,kubectl,helmfile,sops,aws-cli,aws-iam-authenticator,azure-cli,yamllint,updatecli,jenkins-agent,doctl,jq,yq,ruby"
```

```suggestion
      value: "helm,kubectl,helmfile,sops,aws-cli,aws-iam-authenticator,yamllint,updatecli,jenkins-agent,doctl,jq,yq,ruby,azure-cli"
```
See previous comment on Dockerfile